src/commcare_cloud/environment/main.py

```diff
@@ -223,6 +224,16 @@ def proxy_config(self):
         proxy_config.check()
         return proxy_config
 
+    @memoized_property
+    def prometheus_config(self):
+        try:
+            with open(self.paths.prometheus_yml) as f:
+                prometheus_json = yaml.safe_load(f)
+        except IOError:
+            prometheus_json = {}
```

Comment:
If you make any of the fields required then you need can't call `PrometheusConfig.wrap({})` (empty dict). Instead you could just return an empty JsonObject (assuming the only call to it is `.to_json()`
```suggestion
            return JsonObject()
```
This will need to be imported `from jsonobject import JsonObject`